pandora/workers/pdf.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
+
+from __future__ import annotations
+
+import fitz # type: ignore[import-untyped]
+
+from ..helpers import Status
+from ..task import Task
+from ..report import Report
+
+from .base import BaseWorker
+
+class Pdf(BaseWorker):
+
+    check_encoding: bool
+    check_javascript: bool
+    check_suspicious_object: bool
+    check_embedded_files: bool
+
+    def _is_encoding(self, file_path: str) -> bool:
+        try:
+            doc = fitz.open(file_path)
+            return doc.is_encrypted
```

Comment:
Should this method be called `_is_encrypted` instead? Any reason you call the method `_is_encoded`?

Reply:
Thanks!